CONTRIBUTING.md

```diff
@@ -91,12 +91,16 @@ Return types of methods should be specified by comments on the assoc with the pa
 #foo_bar_method
 (declare
     (assoc
+        ; list, a list of strings
```

Comment:
no spaces between ; and comment for any first block of comments, so these 4 in this file need to have that space removed

Reply:
Addressed in 5dbbf38